hr_employee_product/security/product_security.xml

```diff
@@ -14,5 +14,15 @@
             <field name="domain_force">[(1, '=', 1)]</field>
             <field name="groups" eval="[(4,ref('hr.group_hr_manager'))]"/>
         </record>
+
+        <record id="property_rule_product_employee_hr_manager" model="ir.rule">
+            <field name="name">HR Managers can view products that represent employees</field>
+            <field name="model_id" ref="product.model_product_template"/>
+            <field name="groups" eval="[(4,ref('base.group_user'))]"/>
```

Comment:
But this is not HR managers...

Reply:
It's mean adding HR managers Groups  'group_hr_manager' also

Reply:
I still don't understand it...